i18n/README.fr.md

```diff
@@ -0,0 +1,413 @@
+
+- Comment puis-je contribuer ?
+
+  - Créez un problème si vous rencontrez des bugs ou si vous avez des idées de fonctionnalités que nous devrions implémenter. Vous pouvez également ajouter votre propre code en créant ce référentiel et en créant un PR. Plus de détails sur la façon de contribuer avec du code se trouvent dans notre [guide de contribution] (CONTRIBUTING.md).
+
+- Ce framework est-il gratuit ?
+
+  - Oui, ce framework est totalement gratuit et nous fournissons le code source. Nous pensons que nous devons rendre la formation des modèles d’apprentissage profond plus efficace en matière de données pour parvenir à une adoption généralisée. Une étape pour atteindre cet objectif consiste à tirer parti de l’apprentissage auto-supervisé. La société derrière Lightly s'engage à garder ce framework open source.
+
+- Si ce framework est gratuit, comment l'entreprise derrière Lightly gagne-t-elle de l'argent ?
+  - La formation de modèles auto-supervisés n'est qu'une partie de notre solution.
+    [La société derrière Lightly](https://lightly.ai/) se concentre sur le traitement et l'analyse des intégrations créées par des modèles auto-supervisés.
+    En créant ce que nous appelons une boucle d'apprentissage actif auto-supervisé, nous aidons les entreprises à comprendre et à utiliser leurs données plus efficacement.
+    La [Solution Lightly](https://docs.lightly.ai) étant un produit freemium, vous pouvez l'essayer gratuitement. Cependant, nous facturerons certaines fonctionnalités.
+  - Dans tous les cas, ce framework sera toujours libre d'utilisation, même à des fins commerciales.
+
+## Légèrement dans la recherche
+
+- [Apprentissage auto-supervisé par ingénierie inverse, 2023](https://arxiv.org/abs/2305.15614)
+- [Apprentissage des représentations visuelles via un échantillonnage guidé par le langage, 2023](https://arxiv.org/pdf/2302.12248.pdf)
+- [Méthodes d'apprentissage auto-supervisées pour la classification des caries dentaires efficace sur le label, 2022](https://www.mdpi.com/2075-4418/12/5/1237)
+- [DPCL : Apprentissage de représentation contrastive avec confidentialité différentielle, 2022](https://assets.researchsquare.com/files/rs-1516950/v1_covered.pdf?c=1654486158)
+- [Apprentissage contrastif découplé, 2021](https://arxiv.org/abs/2110.06848)
+- [apprentissage en solo : une bibliothèque de méthodes auto-supervisées pour l'apprentissage de la représentation visuelle, 2021](https://www.jmlr.org/papers/volume23/21-1155/21-1155.pdf)
```

Comment:
Please do not translate names and paper titles. This also applies for the rest of the document.

Reply:
same applicable here too